commands/drives.go

```diff
@@ -0,0 +1,133 @@
+package commands
+
+import (
+	"github.com/digitalocean/doctl"
+	"github.com/digitalocean/doctl/do"
+	"github.com/digitalocean/godo"
+
+	"github.com/spf13/cobra"
+)
+
+// Drive creates the Drive command
+// NOTE: This command will currently only work for those in the
+// block storage private beta on DigitalOcean.
+func Drive() *Command {
+	cmd := &Command{
+		Command: &cobra.Command{
+			Use:   "drive",
+			Short: "drive commands",
+			Long:  "drive is used to access drive commands",
+		},
+	}
+
+	CmdBuilder(cmd, RunDriveList, "list", "list drive", Writer,
+		aliasOpt("ls"), displayerType(&drive{}))
+
+	cmdDriveCreate := CmdBuilder(cmd, RunDriveCreate, "create [name]", "create a drive", Writer,
+		aliasOpt("c"), displayerType(&drive{}))
+
+	AddIntFlag(cmdDriveCreate, doctl.ArgDriveSize, 100, "Size of the drive (GiB)",
```

Comment:
I had put 100GB in earlier just to make testing easier. Is it something that we want to move forward with, or do we want to make there not be a default? Open to opinions